scalableminds/webknossos
# Lift restriction that webknossos-worker can only operate on local datastore

save a `jobsEnabled` boolean per datastore. All datastores for which the webknossos-worker has disk read acces can be configured in wk to support jobs.

Note that this does not yet mean a worker can be configured *per datastore*, only one worker can exist currently, it just doesn’t have to be for the local datastore anymore

### Steps to test:
- set up stand-alone datastore among internal datastore
- in db, set each to jobsEnabled true and false, see that dataset json contains corresponding correct values
- In info tab for dataset, the “process dataset” menu item should only appear if the datastore hosting that dataset is marked as jobsEnabled AND features.jobsEnabled is true in the application.conf
- for local development with local datastore, `yarn enable-jobs` and `yarn disable-jobs` were adapted so that they modify the existing localhost datastore entry in the database.

### Issues:
- fixes #5740

------
